docs/modules/nifi/pages/usage_guide/flow-versioning.adoc

```diff
@@ -0,0 +1,74 @@
+= Flow Versioning
+
+IMPORTANT: The way described in this document to version flows is only available in NiFi 2.0.0 and later versions. Previous versions of NiFi relied exclusively on the NiFi Registry service for flow versioning.
+
+Starting with Apache NiFi 2.0.0, you can connect directly to Git repositories using either GitHub or GitLab as your flow registry without requiring a separate NiFi Registry service. This document explains how to configure and use these registry clients to version your NiFi flows.
+
+== Setting up the Registry Client
+
+=== Configuration Steps
+
+1. In your NiFi UI, click on the hamburger menu (☰) in the top right corner
+2. Select "Controller Settings" > "Registry Clients"
+3. Click the "+" icon to add a new registry client
+4. Select "GitHubFlowRegistryClient" or "GitLabFlowRegistryClient" as the client type and click "Add"
```

Comment:
I think quotes are fine for e.g. button references in the UI. This i would rather tick.
```suggestion
4. Select `GitHubFlowRegistryClient` or `GitLabFlowRegistryClient` as the client type and click "Add"
```